### PR TITLE
Fixed broken link

### PR DESCRIPTION
**Pull request by mahesh2492**
- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md)?
- If submitting code, have you built your formula locally prior to submission with `gradle check`?
- If submitting code, is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed.
- If submitting code, have you checked that your submission is for an [OS that we support](https://www.elastic.co/support/matrix#show_os)?
- If you are submitting this code for a class then read our [policy](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md#contributing-as-part-of-a-class) for that.

### CODE REVIEW

File: docs/reference/getting-started.asciidoc [hunk 1]
```diff
@@ -545,7 +545,7 @@ For the curious, I generated this data from http://www.json-generator.com/[`www.
 [float]
 === Loading the Sample Dataset
 
-You can download the sample dataset (accounts.json) from https://github.com/bly2k/files/blob/master/accounts.zip?raw=true[here]. Extract it to our current directory and let's load it into our cluster as follows:
+You can download the sample dataset (accounts.json) from https://raw.githubusercontent.com/elastic/elasticsearch/master/docs/src/test/resources/accounts.json[here]. Extract it to our current directory and let's load it into our cluster as follows:
```
Comment (mayya-sharipova) [inline docs/reference/getting-started.asciidoc]: I make it consistent with other versions, can we make the link
https://github.com/elastic/elasticsearch/blob/master/docs/src/test/resources/accounts.json?raw=true

**Review by mayya-sharipova — CHANGES_REQUESTED** [review]
@mahesh2492  Thank you for your contribution.
Can you please update the link as requested, and also make sure the lines are 80 columns wide (as much as possible)

File: docs/reference/getting-started.asciidoc (hunk 1 above)
Comment (mahesh2492) [reply docs/reference/getting-started.asciidoc]: @mayya-sharipova Thanks for the link. I will update this.

**Comment by mahesh2492** [comment]
@mayya-sharipova I have already updated. Can you look at it?

**Comment by mahesh2492** [comment]
@colings86 If everything seems fine, can you look at this PR?

**Comment by mahesh2492** [comment]
@mayya-sharipova I have updated the PR. Could you please get time to look at it once again?

**Comment by javanna** [comment]
heya @mahesh2492 thanks for opening this, but your PR was made against a very very old branch of Elasticsearch (1.5) which has not received updates for a very long time. In fact the link has been updated in more recent branches. I am closing this as we are not going to push updates to the 1.5 branch at this time.

**Pull request closed without merging**